010-AzureDevOps/Coach/Solution-04.md

```diff
@@ -4,17 +4,11 @@
 
 ## Notes & Guidance
 
-This is the only section you need to include.
+- Use classic version in the ARM deployment
+- Override template parameters, student should have this solution:
+  - Students can archive this item clicking in the 3 dots, open a new window with the name/value table.
+  - Solution tested: -webAppName "updevdevops-dev" -hostingPlanName "updevdevops-asp" -appInsightsLocation "Central US" -appInsightsName "updevdevops-ai" -sku "S1" -registryName "updevdevopsreg" -imageName "updevdevopsimage" -registryLocation "Central US" -registrySku "Standard" -startupCommand ""
+  - check the YAML in the solutions **[armtemplatedeployment.yam](./solutions/armtemplatedeployment.yaml)** for the prefix part
+- Check the **[Solution-04](./solutions/solution-04.json)** for entire possible solution for this challenge. 
```

Comment:
Thanks for updating to make them clickable, however the links are not resolving.  Please note they are case sensitive.